tagger.go

```diff
@@ -170,6 +216,7 @@ func main() {
 	// prep and parse flags
 	flag.String("config", "", "Path to configuration file to use")
 	flag.String("logging.level", "", "Logging level may be one of: trace, debug, info, warning, error, fatal and panic")
+	flag.Bool("report", true, "Optional report output to summarize tag changes")
```

Comment:
If we're going to have a flag that can enable report style output, I think we should probably default it to `false`. Otherwise, why make it a flag?